### PR TITLE
Updating OperationResults Model and examples

### PR DESCRIPTION
**Pull request by ayushiaks**
Adding OperationResults resource under Location path, changing model and examples for the same.
Adding the same in both API versions

### CODE REVIEW

**Review by IshitaAsthana — APPROVED** [review]
Breaking changes are allowed in Private Preview. Have taken permission from AzBreakingChangeReview team

File: specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json [hunk 1]
```diff
@@ -276,16 +256,62 @@
...
       "get": {
-        "tags": [
-          "FarmBeatsExtensions"
+        "tags": ["OperationResults"],
+        "description": "Get operationResults for a FarmBeats resource.",
+        "operationId": "OperationResults_Get",
+        "produces": ["application/json"],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
```
Comment (zizw123) [inline specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json:267]: >v2 [](http://example.com/codeflow?start=69&length=2)

non-blocking, consider updates these to latest version

File: specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json [hunk 2]
```diff
@@ -8,14 +8,10 @@
   "paths": {
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AgFoodPlatform/farmBeats/{farmBeatsResourceName}/extensions/{extensionId}": {
       "put": {
-        "tags": [
-          "Extensions"
-        ],
+        "tags": ["Extensions"],
```
Comment (mikekistler) [inline specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json:11]: Please avoid making superfluous format changes like this -- it just makes reviewing that much more difficult.

File: specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json [hunk 3]
```diff
@@ -276,16 +256,62 @@
         }
       }
     },
-    "/providers/Microsoft.AgFoodPlatform/farmBeatsExtensionDefinitions": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AgFoodPlatform/locations/{locations}/operationResults/{operationResultsId}": {
```
Comment (mikekistler) [inline specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json:259]: You shouldn't need to make changes to this API as you should have moved all customers to the more recent 2021-09-01-preview and retired this one 90 days after the availability of that release.

Comment (ayushiaks) [reply specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json:279]: We haven't yet deprecated/retired our existing version yet. We will be working on it in sometime.

File: specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json (hunk 2 above)
Comment (ayushiaks) [reply specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json]: Fixed this, ran prettier and it worked fine

File: specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json (hunk 1 above)
Comment (ayushiaks) [reply specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json:291]: yes definitely, that is in our ToDos.

**Comment by weidongxu-microsoft** [comment]
@qiliao123 @tadelesh @msyyc for awareness of the breaks (that should be in preview)

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**